src/app/clusters/scenes/SceneTableImpl.cpp

```diff
@@ -283,7 +283,7 @@ struct FabricSceneData : public PersistentData<kPersistentFabricBufferMax>
 
         uint8_t i = 0;
         CHIP_ERROR err;
-        uint8_t deleted_scenes_count = 0;
```

Comment:
Should probably still set it to 0 up front.

Reply:
Added.